electron/main.ts

```diff
@@ -368,6 +378,9 @@ ipcMain.handle('install', async (event, args) => {
 })
 
 ipcMain.handle('repair', async (event, game) => {
+  if (!(await isOnline())) {
+    console.log(`App offline, skipping pair for game '${game}'.`)
```

Comment:
With this is still trying todo the repair right?

Reply:
repair is often called by a user interaction. Maybe its an idea to show a warning to the user that tell them why there is nothing happening? Like a dialog that tell them they can not repair the game because they appear to be offline.

Reply:
Thats true. Sometimes after the repair legendary starts downloading the files that are missing or damaged, so while offline this will throw an error and would be good to show something to the user.

Reply:
> With this is still trying todo the repair right?

Oh I'm a total idiot, forgot to actually return after the log.